providers/song-info.js

```diff
@@ -1,73 +1,20 @@
+const { ipcMain } = require("electron");
```

Comment:
nit: the 2 electron imports could be bundled in one

Reply:
Oh whoops, didn’t spot this one